examples/calendar/backend/privacy/proof.sh

```diff
@@ -0,0 +1,12 @@
+circom calendar.circom --r1cs --wasm --sym --c
```

Comment:
which of the added files you need to add and which are generated with this script? if some of generated files are redundant please remove them and add gitignore
same for script below

Reply:
I will make a note of it.

Reply:
All the files in `privacy_cir/` except `caledar.circom`, `input.json`, `setup.sh` and `proof.sh` are auto generated.